### PR TITLE
Output format changes

### LINKED ISSUE #61: Extend information offered by /forSignature endpoint about matched signatures
For now `/forSignature` endpoint in `HttpServer` sends String as a list of prettified matched signatures. This should be changed to a proper json with fields including e.g. (prettified) signature, function name, package. This external type should be declared in `inkuire-common`, so that the inkuire-intellij-plugin will be able to use it as well.

### PR DESCRIPTION
**Pull request by pikinier20**
#closes #61

### CODE REVIEW

File: inkuire-engine/src/main/scala/org/virtuslab/inkuire/engine/http/OutputFormatter.scala [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package org.virtuslab.inkuire.engine.http
+
+import org.virtuslab.inkuire.engine.model.ExternalSignature
+import org.virtuslab.inkuire.engine.service.SignaturePrettifier
+import org.virtuslab.inkuire.model.{Match, OutputFormat}
+import collection.JavaConverters._
+
+class OutputFormatter(val prettifier: SignaturePrettifier) {
```
Comment (KacperFKorban) [inline inkuire-engine/src/main/scala/org/virtuslab/inkuire/engine/http/OutputFormatter.scala:8]: rm val

Comment (pikinier20) [reply inkuire-engine/src/main/scala/org/virtuslab/inkuire/engine/http/OutputFormatter.scala:8]: It's not `case class`

Comment (KacperFKorban) [reply inkuire-engine/src/main/scala/org/virtuslab/inkuire/engine/http/OutputFormatter.scala:8]: I know

Comment (KacperFKorban) [reply inkuire-engine/src/main/scala/org/virtuslab/inkuire/engine/http/OutputFormatter.scala:8]: It's not Kotlin

Comment (pikinier20) [reply inkuire-engine/src/main/scala/org/virtuslab/inkuire/engine/http/OutputFormatter.scala]: Removed

**Review by KacperFKorban — APPROVED** [review]

**Pull request merged**